BoneCentral/Brain/Utilities/compassTools.js

```diff
@@ -0,0 +1,51 @@
+var vector = require('./vector.js');
+
+var getTrueCompass = function(compv, accelv) {
+    var av      = unitScale(accelv);
+    var right   = unitScale(vector.cross(av, compv));
+    var res     = unitScale(vector.cross(right, av));
+    return res;
+};
+
+var cdeg = function(compv, va) {
```

Comment:
Also I specifically mean the names of the variables in the function cdeg and the function name itself.  I kind of failed miserably and selecting the code segment.